packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -30,7 +30,9 @@ export const messageThreadContainerStyle = mergeStyles({
   width: '100%',
   maxHeight: '100%',
   overflow: 'auto',
-  position: 'relative'
+  position: 'relative',
+  alignSelf: 'center',
+  maxWidth: '41.25rem'
```

Comment:
If we don't have maxWidth what is the experience like? Does it get **too wide** ? (I've personally only had issues when it gets too small but not too large)

Reply:
Yes the maxWidth is just for when things get too wide. Lets chat with Alexp about these